pkg/client/saramaclient.go

```diff
@@ -49,6 +56,48 @@ func (s *SaramaClient) ListBrokers() map[int]string {
 	return brokerMap
 }
 
+func (s *SaramaClient) ListConsumerGroups() (map[string]string, error) {
+	return s.admin.ListConsumerGroups()
+}
+
+func (s *SaramaClient) GetConsumerGroupsForTopic(groups []string, topic string) (chan string, error) {
+	var wg sync.WaitGroup
+	consumerGroupsChannel := make(chan string, len(groups))
+
+	if len(topic) == 0 {
```

Comment:
1. we could return error if topic is empty
2. prefer range over regular for loop when you don't need the index